Code/Framework/AzToolsFramework/AzToolsFramework/UI/DocumentPropertyEditor/DocumentPropertyEditor.cpp

```diff
@@ -230,10 +345,30 @@ namespace AzToolsFramework
         return theDPE;
     }
 
+    DPERowWidget* DPERowWidget::GetLastDescendentInLayout()
+    {
+        DPERowWidget* lastDescendent = nullptr;
```

Comment:
uber-nit: descend_a_nt

(Looked it up online, sounds like descendent is not incorrect but is mostly used as an adjective, while descendant is the most common spelling)